source/log4net-loggly/LogglyFormatter.cs

```diff
@@ -109,7 +109,16 @@ private string GetMessageAndObjectInfo(LoggingEvent loggingEvent, out object obj
             objInfo = null;
             var bytesLengthAllowedToLoggly = EVENT_SIZE;
 
-            if (loggingEvent.MessageObject != null)
+            if (!string.IsNullOrEmpty(loggingEvent.RenderedMessage))
```

Comment:
I looked into the details and found the below for **LogglyEvents.MessageObject** property-
 **"Gets the message object used to initialize this event. Note that this event may not have a valid message object. If the event is serialized the message object will not be transferred. To get the text of the message the RenderedMessage property must be used not this property."** from [here](https://logging.apache.org/log4net/release/sdk/html/P_log4net_Core_LoggingEvent_MessageObject.htm). 

I debugged the code here and could see that each time only the first **if block** was getting executed because we have the value in **loggingEvent.RenderedMessage**. The else block to check **loggingEvent.MessageObject** is not required anymore so I have removed it after ensuring. 

And this is not specific to dot net core, it's a common use case for old .NET Frameworks too.